front-end/src/screens/Home/Home.tsx

```diff
@@ -103,26 +45,90 @@ const Home = ({ data }: Props) => {
 								),
 						)}
 					</ul>
-				</Col>
-				<Col md={6} lg={5}>
-					{currentUser.id && <Button className={'newPostButton'} onClick={handleCreatePost}>New Post</Button>}
+				</Grid.Column>
+				<Grid.Column mobile={16} tablet={16} computer={8}>
 					<div className='info_box'>
-						<h5>Hello</h5>
+						<h4>About Polkassembly</h4>
 						<div className='info_text'>Discuss all things Polkadot governance, signal your intentions, back on-chain proposals and vote on referenda.</div>
-						<div className='info_text'><h6>New to Polkadot Governance?</h6> Polkadot uses a sophisticated governance mechanism to let the network evolve gracefully. The stated goal is to ensure that the majority of the stake can always command the network.</div>
-						<div className='ex_link'>
-							<a
-								href='https://wiki.polkadot.network/docs/en/learn-governance'
-								rel='noopener noreferrer'
-								target='_blank'>
-								Find out more about Polkadot Governance »
-							</a>
+						<div className='mainButtonContainer'>
```

Comment:
I really don't think the info box is the right place to put a "new post" button.
We should keep this button at the top.
Also this info box will most probably move away as soon as we have proposals and vote.